components/ProjectList.js

```diff
@@ -0,0 +1,26 @@
+import Project from "./Project";
+import Error from "@/components/Error";
+import useSWR from "swr";
+
+export default function ProjectList() {
+  const { data: projects, isLoading, error } = useSWR("/api/projects");
+
+  if (isLoading) {
+    return <h1>Loading ... </h1>;
```

Comment:
please only use h1 once overall your page (in the header)

Reply:
thanks Julia. we are on it now